test/test_proxy_tensor.py

```diff
@@ -1946,75 +2234,89 @@ def f(t):
 
 make_fx_failures = {
     # unknown
-    xfail('allclose'),
-    xfail('equal'),
+    xfail("allclose"),
+    xfail("equal"),
     # empty
-    skip('new_empty'),
-    skip('empty_like'),
-    skip('empty'),
-    skip('empty_permuted'),
+    skip("new_empty"),
+    skip("empty_like"),
+    skip("empty"),
+    skip("empty_permuted"),
     # flaky
-    skip('linalg.lstsq', 'grad_oriented'),
-    skip('nn.functional.max_unpool1d', '', device_type='cpu'),
-    skip('nn.functional.max_unpool2d', '', device_type='cpu'),
-    skip('nn.functional.max_unpool3d', '', device_type='cpu'),
-    skip('linalg.lstsq'),  # flaky, probably just a precision issue
-
+    skip("linalg.lstsq", "grad_oriented"),
+    skip("nn.functional.max_unpool1d", "", device_type="cpu"),
+    skip("nn.functional.max_unpool2d", "", device_type="cpu"),
+    skip("nn.functional.max_unpool3d", "", device_type="cpu"),
+    skip("linalg.lstsq"),  # flaky, probably just a precision issue
     # data-dependent control flow
-    skip('item'),
-    xfail('cov'),
-    xfail('nn.functional.gaussian_nll_loss'),
-    xfail('corrcoef'),
-    xfail('quantile'),
-    xfail('nanquantile'),
-
+    skip("item"),
+    xfail("cov"),
+    xfail("nn.functional.gaussian_nll_loss"),
+    xfail("corrcoef"),
+    xfail("quantile"),
+    xfail("nanquantile"),
     # Seems like it's creating a sparse tensor that isn't captured by tensor.is_sparse
-    xfail('sparse.sampled_addmm'),
-    xfail('sparse.mm', 'reduce'),
-
+    xfail("sparse.sampled_addmm"),
+    xfail("sparse.mm", "reduce"),
     # proxy tensor doesn't support sparse correctly right now
-    skip('to_sparse'),
+    skip("to_sparse"),
     # segfaults
-    skip('block_diag'),
-
+    skip("block_diag"),
     # AssertionError: Tensor-likes are not close!
-    skip('empty_strided', '', device_type='cpu'),
+    skip("empty_strided", "", device_type="cpu"),
 }
 
 only_real_tensor_failures = {
     xfail('narrow'),
-    xfail('tensor_split'),
+    xfail("tensor_split"),
 }
 
 only_fake_tensor_failures = {
-    xfail('narrow'),
```

Comment:
This is the only substantial change I could find in this file - correct me if wrong :)

Reply:
right but this is k we do not fail now.